mozilla_django_oidc/utils.py

```diff
@@ -51,3 +57,46 @@ def is_authenticated(user):
     msg = '`is_authenticated()` is going to be removed in mozilla-django-oidc v 2.x'
     warnings.warn(msg, DeprecationWarning)
     return user.is_authenticated
+
+
+def add_state_and_nonce_to_session(request, state, params):
+    nonce = None
+    if import_from_settings('OIDC_USE_NONCE', True):
```

Comment:
It would be better to leverage the staticmethod `get_settings` which can be used to configure the OIDC lib for multiple providers.

Reply:
@akatsoulas: There is no `get_settings` method in the `utils.py` file. And when I would use one of the existing implemented `get_settings` methods, we would have a circular dependency.
Any suggestions?

Reply:
I used the method like in `views.py` `get_next_url` or in `urls.py`.